third-party/CMakeLists.txt

```diff
@@ -116,8 +116,13 @@ endif()
 
 include(ExternalProject)
 
-set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.12.0.tar.gz)
-set(LIBUV_SHA256 41ce914a88da21d3b07a76023beca57576ca5b376c6ac440c80bc581cbca1250)
+if(WIN32)
+  set(LIBUV_URL https://github.com/neovim/libuv)
+  set(LIBUV_TAG nvim)
```

Comment:
We could use:

    set(LIBUV_URL https://github.com/neovim/libuv/archive/c442232790cbcd82da80356c931a8921dc308115.tar.gz)
    set(LIBUV_SHA256 ...)

Then we don't need the changes to `BuildLibuv.cmake` (right?). And this is preferred anyways, because it's important to have "reproducible build" when bisecting.

Reply:
@justinmk I did not know that we can download the tarball with the commit ID specified. After completing the work, I was planing to have you do create tag.